regtest/Dockerfile.bchd

```diff
@@ -2,8 +2,8 @@
 # and a workspace (GOPATH) configured at /go.
 FROM golang:1.15
 
-RUN apt update \
-    && apt install -y dnsutils
+RUN apt-get update \
+    && apt-get install -y dnsutils
```

Comment:
This is because `apt` is for human consumption while `apt-get` is for scripts.

Reply:
TIL!